loaders/httpfs/loader.go

```diff
@@ -0,0 +1,30 @@
+package httpfs
+
+import (
+	"io"
+	"net/http"
+)
+
+type httpFileSystemLoader struct {
+	http.FileSystem
+}
+
+func NewLoader(fs http.FileSystem) httpFileSystemLoader {
```

Comment:
hmm, that's also not possible

`eval_test.go` uses some unexported fields from the `jet` pkg. So this file must remain in `jet` pkg.

And I can't simply move the `httpfs` tests to a different pkg, since it's using functions like `RunJetTestWithSet()`.